### PR TITLE
PR:Typography

### PR DESCRIPTION
**Pull request by Jayesh3636**
- Added typography
  - Added Heading text class
  - Added Paragraph text class
  - Added Aligned Paragraph text class
  
  
Kindly review the component and let me know the changes! TIA!
 

https://user-images.githubusercontent.com/91378157/153714475-69ca6469-e2fb-4226-9eb1-4429844b684c.mp4

### CODE REVIEW

**Review by TusharJade — COMMENTED** [review]
UI is Awesome, Good to go👍

File: Components/typography/typography.css [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+    color: var(--primary-color)
+}
+/* span class blue */
+
+.iframe{
+    
+    box-sizing: border-box;
+    width: 100%;
+    padding: 1rem 0;
+    background-color: #2F1E2E;
```
Comment (shubhmehra) [inline Components/typography/typography.css]: please use color from your root

File: Components/typography/typography.css [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+        padding: 3rem;
+        background-color: rgb(255, 245, 245);
+        border-radius: 12px;
+        color: #5b5b5b;
+        text-align: center;
+        transition: all ease-in-out 300ms;
+    }
+    
+    .examples-box-align:hover {
+        background-color: rgb(27, 27, 27);
```
Comment (shubhmehra) [inline Components/typography/typography.css]: same try to use rgb & hex colors from root

File: Components/typography/typography.html [hunk 3]
```diff
@@ -0,0 +1,230 @@
...
+                                <h3 class="text-right">Heading-right <span class="blue-text">3</span></h3>
+                            </div>
+                        </div>
+
+                        <div class="iframe">
+                            <iframe
+                                src="https://carbon.now.sh/embed?bg=rgba%28255%2C255%2C255%2C0%29&t=paraiso-dark&wt=none&l=css&width=680&ds=true&dsyoff=0px&dsblur=0px&wc=true&wa=false&pv=0px&ph=0px&ln=false&fl=1&fm=Droid+Sans+Mono&fs=14px&lh=209%25&si=false&es=2x&wm=false&code=%253Ch3%2520class%253D%2522text-left%2522%253EHeading-left%2520%253Cspan%2520class%253D%2522blue-text%2522%253E1%253C%252Fspan%253E%253C%252Fh3%253E%250A%253Ch3%2520class%253D%2522text-center%2522%253EHeading-center%2520%253Cspan%2520class%253D%2522blue-text%2522%253E2%253C%252Fspan%253E%253C%252Fh3%253E%250A%253Ch3%2520class%253D%2522text-right%2522%253EHeading-right%2520%253Cspan%2520class%253D%2522blue-text%2522%253E3%253C%252Fspan%253E%253C%252Fh3%253E"
+                                style="width: 680px; height: 145px; border:0; transform: scale(1); overflow:hidden;"
+                                sandbox="allow-scripts allow-same-origin">
```
Comment (shubhmehra) [inline Components/typography/typography.html:194]: use 100% to make iframe responsive

File: Components/typography/typography.html [hunk 4]
```diff
@@ -0,0 +1,230 @@
...
+
+
+
+
+
+
+
+
+
+
```
Comment (shubhmehra) [inline Components/typography/typography.html]: you can remove this blank line & reduce line of code in file

File: Components/typography/typography.html (hunk 3 above)
Comment (Jayesh3636) [reply Components/typography/typography.html:194]: done

File: Components/typography/typography.css (hunk 2 above)
Comment (Jayesh3636) [reply Components/typography/typography.css]: done

File: Components/typography/typography.html (hunk 4 above)
Comment (Jayesh3636) [reply Components/typography/typography.html]: done

**Pull request merged**